packages/backend/services/connection.ts

```diff
@@ -29,11 +29,7 @@ class ConnectionService {
         const { 'x-revert-api-token': token } = req.headers;
         const connections: any = await prisma.connections.findMany({
             where: {
-                account: {
-                    is: {
```

Comment:
@Hvinder This is a bug. We'd need to use the private token here instead of the public token.